src/imitation/data/types.py

```diff
@@ -339,3 +515,22 @@ def __post_init__(self):
         """Performs input validation, including for rews."""
         super().__post_init__()
         _rews_validation(self.rews, self.acts)
+
+
+ObsType = TypeVar("ObsType", np.ndarray, DictObs)
+
+
+def concatenate_maybe_dictobs(arrs: List[ObsType]) -> ObsType:
```

Comment:
Maybe these should move into the DictObjs class.

Reply:
I'm generally fine with module-level helper methods in Python, we can always make them private if not used outside of this module.